### PR TITLE
HELP-25940: kazoo updates for cdr summary API

### PR DESCRIPTION
**Pull request by k-anderson**

### CODE REVIEW

**Comment by k-anderson** [comment]
Please do not merge yet, requires an update to couchbeam to work properly.

File: applications/crossbar/doc/ref/cdrs.md [hunk 1]
```diff
@@ -72,6 +72,16 @@ curl -v -X GET \
 
 #### Fetch
 
+> GET /v2/accounts/{ACCOUNT_ID}/cdrs/summary
```
Comment (jamesaimonetti) [inline applications/crossbar/doc/ref/cdrs.md]: Looks like a new endpoint path that should be added to the [doc](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/cdrs.md), eh?

**Comment by fenollp** [comment]
Where are we on the couchbeam patch?
BTW it looks like we're using an outdated version of couchbeam: https://github.com/lazedo/couchbeam/compare/master...1.3.1.2

**Comment by lazedo** [comment]
proper compare https://github.com/lazedo/couchbeam/compare/1.3.1.2...benoitc:master

**Pull request merged**